asn1/InterledgerProtocol.asn

```diff
@@ -0,0 +1,31 @@
+InterledgerProtocol
+DEFINITIONS
+AUTOMATIC TAGS ::=
+BEGIN
+
+IMPORTS
+    UInt64
```

Comment:
I was thinking about that. If there is an amount type, we should use it everywhere obviously. But if `Amount` is just `UInt64` I think it's more readable if we just put `UInt64` everywhere. For complex types, sure, define it once. But if it's just a primitive type, don't make me go to another file just to find out what it is. What do you think?